include/libopencm3/sam/4l/pm.h

```diff
@@ -26,63 +26,53 @@
 
 /* Main Clock Control MCCTRL Read/Write */
 #define PM_MCCTRL			MMIO32(PM_BASE + 0x000)
+#define PM_MCCTRL_KEY		(PM_UNLOCK_KEY)
 
-/* CPU Clock Select CPUSEL Read/Write */
-#define PM_CPUSEL			MMIO32(PM_BASE + 0x004)
```

Comment:
Well, if I never made mistakes, I would not be using version control to begin with.

I understand that you are trying enforce "everything is consistent and works at every commit" rule, but the problem with rebase is that it completely borkes the branch, if it is already on github. That is, if I do rebase on this branch at home and then try to push changes to github (where this branch already exists), the results are ... well, funny. I basically had to create a new branch and make a new pull request (see "2" at the end of my branch name -- had to do it twice).

This is OK, but what about my responses to your comments? When I make a fix and push it -- I create new commits that fix earlier commits that are not merged into your main branch yet. So, do I have to rebase and make a new pull request every time? This does not sound like a very sustainable model.